README.md

```diff
@@ -25,7 +25,7 @@ To install the extension on an existing CiviCRM site:
 mkdir sites/all/modules/civicrm/ext
 cd sites/all/modules/civicrm/ext
 git clone https://github.com/civicrm/org.civicrm.shoreditch shoreditch
-git clone https://github.com/civicrm/org.civicrm.civicase civicase
+git clone https://github.com/civicrm/uk.co.compucorp.civicase civicase
```

Comment:
The URL is wrong, should be `https://github.com/compucorp/uk.co.compucorp.civicase.git`